src/main/java/emissary/core/BaseDataObject.java

```diff
@@ -448,6 +451,9 @@ public int enqueueCurrentForm(final String newForm) {
     public int pushCurrentForm(final String newForm) {
         if (newForm == null) {
             throw new IllegalArgumentException("caller attempted to push a null form value");
+        } else if (newForm.indexOf(KeyManipulator.SEPARATOR) > 0) {
```

Comment:
Should this case also throw an ```IllegalArgumentException```?

Reply:
This issue will more than likely be discovered in a unit test with proper code coverage if an exception is thrown.

Reply:
> Should this case also throw an `IllegalArgumentException`?

The upstream ticket indicates a plan to only log these in the short term, in order _(I'm assuming)_ to quantity the frequency of occurrence and to determine if an exception here would cause more harm than good.

Reply:
Is the intention to not log if ```newForm``` start with a separator? If not, then maybe consider ```StringUtils.contains(newForm, KeyManipulator.SEPARATOR)```